crates/cxx-qt-build/src/lib.rs

```diff
@@ -304,8 +305,15 @@ impl CxxQtBuilder {
 
     /// Link additional [Qt modules](https://doc.qt.io/qt-6/qtmodules.html).
     /// Specify their names without the `Qt` prefix, for example `"Widgets"`.
-    /// The Core and Gui modules are linked automatically; there is no need to specify them.
-    pub fn qt_modules(mut self, modules: &[&str]) -> Self {
+    /// The Core and any feature enabled modules are linked automatically; there is no need to specify them.
+    pub fn qt_module(mut self, module: &str) -> Self {
+        self.qt_modules.insert(module.to_owned());
+        self
+    }
+
+    /// Replace the list of [Qt modules](https://doc.qt.io/qt-6/qtmodules.html) to link.
+    pub fn set_qt_modules(mut self, modules: &[&str]) -> Self {
```

Comment:
It was so that you could opt out of Gui before as some projects might want to just link Core.

This is now sortof possible with the feature in cxx-qt-build, except if you put your core-only lib in the same workspace as your gui-enabling lib :thinking: 

So i think i'll move this back to `qt_module` for now.

Reply:
Done